superset/views/base.py

```diff
@@ -96,6 +98,20 @@ def json_error_response(
     )
 
 
+def json_errors_response(
+    errors: List[SupersetError], status: int = 500, payload: Optional[dict] = None,
```

Comment:
Could we type the `dict` i.e., `Dict[..., ...]`.

Reply:
yup, also changing the type for `json_error_response` too to match